src/test/java/org/fundacionjala/trello/stepdefs/CardStepDefs.java

```diff
@@ -36,7 +31,7 @@ public CardStepDefs(final SharedDriver sharedDriver, final Context context) {
      * Creates a card with specific data.
      *
      * @param listName expected name of the list where card will be created.
-     * @param data expected card data.
+     * @param data     expected card data.
```

Comment:
Unnecesarry spaces here

Reply:
This is the configuration forma of check stile